pkg/api/router.go

```diff
@@ -85,6 +85,11 @@ func (s *server) setupRouting() {
 		),
 	})
 
+	handle(router, "/pss/{topic}/ws", jsonhttp.MethodHandler{
```

Comment:
This should be (POST,DELETE) `/pss/subscribe/{topic}` for subscriptions and there should be `/pss/send/{topic}` for sending.

Reply:
cool. i think `DELETE` is not necessary since the client can just hang-up the websocket, or the server can also close the websocket on shutdown. cc @zelig

Reply:
Right, good point @acud.